src/EditorFeatures/CSharpTest/Diagnostics/UpgradeProject/UpgradeProjectTests.cs

```diff
@@ -212,20 +191,19 @@ public class Program
 }
 ",
                 LanguageVersion.CSharp7_1,
-                new CSharpParseOptions(LanguageVersion.CSharp7),
-                index: 0);
+                new CSharpParseOptions(LanguageVersion.CSharp7));
         }
 
-#region C# 7.3
+        #region C# 7.3
```

Comment:
Nit: We have the same problem in compiler code. I thought our preference was to leave regions left-aligned. Can you revert this?

Reply:
➡️ I was going to revert this change since it was a formatting change outside the intended scope of the PR, but since the current form matches the indentation of the `#endregion` and no other changes are required to the PR, I'm going to leave it.